specs/Menu.md

```diff
@@ -1,731 +1,540 @@
 
-A menu is a way of displaying items that a user may be able to interact with. Menus vary quite a bit based on implementation and usage. Some are horizontal, some have collapsable sections, and others are just a list. Likewise menu items are equally varied. Some may just convey information but cannot be interacted with like headers or dividers while others are either buttons, links, or expand into a submenu.
+The `Menu` should be displayed on a temporary surface that interrupts the normal flow of content. The temporary surface should be triggered by an external user action such as (but not limited to) a click on a button or other UI control.
 
-The w3 specifications for a menu suggest that it is only for user actions, not user input. Most varieties of menus interpret this as being for navigation.
+The interactions that result in the dismiss/removal of the `Menu` component should be configurable.
 
-Menus are often combined with a popover or similar floating component so the menu appears over other items and is subsequently dismissed.
+## Prior art
 
-### MenuItem
+As a part of the spec definitions in Fluent UI, a research effort has been made through [Open UI](https://open-ui.org/). The current research proposal is available as an open source contribution undergoing review ([LINK to research proposal](https://github.com/WICG/open-ui/pull/249))
 
-Most menus have a concept of a specific menuitem. It is an integral part of the menu and will have some of its own sections throughout.
+## Comparison of `@fluentui/react` and `@fluentui/react-northstar`
 
-## References
+All mentions of v7 or v8 == `@fluentui/react`
+All mentions of v0 == `@fluentui/react-northstar`
 
-[CodeSandbox of implemented versions](https://codesandbox.io/s/menuexamples-uybsr)
+The most relevant comparison that can be achieved between the two libraries is between `ContextualMenu` in v7 and a combination of `Menu`, `Popup` and `ToolbarItem` in v0.
 
-- [Aria practices](https://www.w3.org/TR/wai-aria-1.1/#menu)
-- [Fabric](https://developer.microsoft.com/en-us/fabric#/controls/web/contextualmenu)
-- [Stardust](https://microsoft.github.io/fluent-ui-react/components/menu/definition)
-- [Material-ui](https://material-ui.com/components/menus/)
-- [Chakra-ui](https://chakra-ui.com/menu)
-- [AntDesign](https://ant.design/components/menu/)
-- [Jquery-ui](https://jqueryui.com/menu/)
-- [Carbon-ui](https://www.carbondesignsystem.com/components/overflow-menu/code)
-- [Fast-DNA](https://github.com/microsoft/fast-dna/blob/master/packages/fast-components-react-base/src/context-menu/context-menu.tsx)
-- [BluePrintJs](https://blueprintjs.com/docs/#core/components/menu)
+v0 suffers from a consistency issue that the control used in `Menu` and the menu variant of `ToolbarItem` are not actually the same component and have different behaviour. However, semantically for the purposes of this spec, they represet the same control that will be implemented.
```

Comment:
```suggestion
v0 suffers from a consistency issue that the control used in `Menu` and the menu variant of `ToolbarItem` are not actually the same component and have different behavior. However, semantically for the purposes of this spec, they represent the same control that will be implemented.
```

typos 🤹

Reply:
I would argue to the death that `behaviour` is the correct spelling, but sure I can change :-)